tests/roles/backend_services/tasks/main.yaml

```diff
@@ -82,6 +82,17 @@
   retries: 60
   delay: 2
 
+- name: wait for mariadb to start up
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc get pod mariadb-openstack-cell1 -o jsonpath='{.status.phase}{"\n"}' | grep Running
+  register: mariadb_running_result
+  until: mariadb_running_result is success
+  retries: 60
+  delay: 2
```

Comment:
so this is timing out in ci which implies that my template changes are not working or i need to change it somewhere else.